docs/src/interface.md

```diff
@@ -64,3 +57,49 @@ julia> y[1]
 
 * [`allocate_result`](@ref) allocates a result of a particular function (for example [`exp`], [`flat`], etc.) on a particular manifold with particular arguments.
   It takes into account the possibility that different arguments may have different numeric [`number_eltype`](@ref) types thorough the [`ManifoldsBase.allocate_result_type`](@ref) function.
+
+## A Decorator for manifolds
+
+ decorator manifold extends the functionality of a [`Manifold`](@ref) in a semi-transparent way.
```

Comment:
```suggestion
A [`DecoratorManifold`](@ref ManifoldsBase.DefaultManifold) extends the functionality of a [`Manifold`](@ref) in a semi-transparent way.
```

Reply:
Why do you want to link the decorator manifold to the `DefaultManifold`?

Reply:
Sorry, that's a mistake. It should be a link to `DecoratorManifold`.

Reply:
Sorry, let's just add the letter 'A' and call it good enough 🙂 .